docs/conf.py

```diff
@@ -66,6 +67,12 @@
     "mdtraj": ("https://www.mdtraj.org/1.9.5/", None),
 }
 
+mermaid_init_js = """mermaid.initialize({
+    startOnLoad:true,
+    fontFamily:"montserrat,sans",
+    theme: "default",
+});"""
+
```

Comment:
That's right. There is one other trick which is that the mermaid block in HTML is whitespace delimited, so tools that automatically minify or otherwise modify it can break them (ask me how I know)